src/nvim/message.c

```diff
@@ -3628,26 +3585,19 @@ int vim_vsnprintf(char *str, size_t str_m, char *fmt, va_list ap, typval_T *tvs)
             }
 
             if (tp != NULL && !precision_specified)
```

Comment:
Well, yes. There are many places where comparison to NULL is used. I'm only changing the ones that are in the places I touch, though.